roles/ceilometer-control/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: restart ceilometer services
+  service: name={{ item }} state=restarted must_exist=false
+  when: restart|default('True')
+  with_items:
+    - ceilometer-api
+    - ceilometer-collector
+    - ceilometer-agent-notification
+    - ceilometer-agent-central
+    - ceilometer-alarm-evaluator
+    - ceilometer-alarm-notifier
```

Comment:
The reason I have these separated is because these 6 services are only on the controller nodes, while the compute only has one service. So merging them all into one handler on the ceilometer-common role would throw an error, right?